src/test/datascience/dataScienceIocContainer.ts

```diff
@@ -617,6 +624,7 @@ export class DataScienceIocContainer extends UnitTestIocContainer {
             ExportManagerDependencyChecker,
             ExportManagerDependencyChecker
         );
+        this.serviceManager.addSingleton<IExportUtil>(IExportUtil, ExportUtil);
```

Comment:
```suggestion
        this.serviceManager.addSingleton<ExportUtil>(ExportUtil, ExportUtil);
```

I'd dump the interface